nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -402,7 +407,7 @@ public Description matchCompoundAssignment(CompoundAssignmentTree tree, VisitorS
       return Description.NO_MATCH;
     }
     Type lhsType = ASTHelpers.getType(tree.getVariable());
-    Type stringType = state.getTypeFromString("java.lang.String");
+    Type stringType = JAVA_LANG_STRING_TYPE.get(state);
```

Comment:
Didn't check out the code, but IIUC in this case one can also reuse an existing catched type:
```suggestion
    Type stringType = state.getSymtab().stringType;
```

Reply:
Thanks for the suggestion!  I went with a built-in supplier from EP just to make the code a bit more readable in 4804639